e2e/app/page/workspace-analysis-page.ts

```diff
@@ -70,8 +70,8 @@ export default class WorkspaceAnalysisPage extends WorkspaceBase {
       this.page.waitForXPath(redirectingTextsXpath, {visible: true}),
     ]);
 
-    // Waiting up to 15 minutes
-    await waitWhileLoading(this.page, (15 * 60 * 1000));
+    // Waiting up to 20 minutes
+    await waitWhileLoading(this.page, (20 * 60 * 1000));
```

Comment:
waiting for notebook start timed out after 15 min. 
https://app.circleci.com/pipelines/github/all-of-us/workbench/6390/workflows/0b93062a-ac0e-4ec1-a0cc-e360d93b0fb9/jobs/113590